platform/common/core/generic/vault/fver/version.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package fver
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/hyperledger-labs/fabric-smart-client/pkg/utils/errors"
+	"github.com/hyperledger-labs/fabric-smart-client/platform/common/driver"
+)
+
+var zeroVersion = []byte{0, 0, 0, 0, 0, 0, 0, 0}
```

Comment:
not sure we can assume that the length will be always 8 unless we change the type. I would change the check `bytes.Equal(zeroVersion, b)` to `collections.ContainsOnly(b, 0)`